src/common/datetime/format_date.ts

```diff
@@ -9,3 +9,12 @@ export const formatDate = toLocaleDateStringSupportsOptions
         day: "numeric",
       })
   : (dateObj: Date) => format(dateObj, "longDate");
+
+export const formatDateWeekday = toLocaleDateStringSupportsOptions
+  ? (dateObj: Date, locales: string) =>
+      dateObj.toLocaleDateString(locales, {
+        weekday: "long",
+        month: "short",
+        day: "numeric",
+      })
+  : (dateObj: Date) => format(dateObj, "longDate");
```

Comment:
This is not correct, check the right format in the docs of fecha <https://github.com/taylorhakes/fecha#formatting-tokens>